libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/internal/NavigationViewApiImpl.kt

```diff
@@ -0,0 +1,36 @@
+package com.mapbox.navigation.dropin.internal
```

Comment:
Shouldn't this live outside internal package, as this is the facade to communicate with NavigationView and invoke API(s) on it?

Reply:
`NavigationViewApi` is the public interface. That, does not have to be an interface. We could delete it and make a facade. Simple functions so, that could be better

Reply:
I think I agree. Looking at it more closely, it doesn't look like we need the interface at all. Since the idea is to not have external implementations, we could just have `NavigationViewApi` as a `class` and host the public API(s). Wdyt?

@tomaszrybakiewicz

Reply:
Updated to `NavigationViewApi` as a class, no interface.